packages/libnvidia-container/libnvidia-container.spec

```
@@ -68,6 +69,7 @@ export DESTDIR=%{buildroot} \\\
 %files
 %license NOTICE LICENSE
 %{_cross_attribution_file}
+%{_cross_libdir}/libnvidia-container.so
 %{_cross_libdir}/libnvidia-container.so.1
```

nit - to avoid the need to revisit this if the so major version is ever updated:
```suggestion
%{_cross_libdir}/libnvidia-container.so.*
```